Automattic/wp-calypso
# Editor Welcome: Add new slide to the Welcome Tour (EN only!)

# NOTE:  D72162-code has to be approved and merged before merging this PR!
 
### Changes proposed in this Pull Request

* Add a new slide "Find your way" to the Editor Welcome Tour

#### Technical changes

In `apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.js`:
* Update tour assets config.
* Update Reference Elements config.
* Update Tour Steps config.

### Testing instructions

**Preparations**

1. Make sure that you're sandboxed
2. Apply D72162-code w/ `arc patch-image D72162` (see PCYsg-h9w-p2)
3. Make sure that s0.wp.com is sandboxed
4. Check out this branch and `yarn && yarn dev --sync`.

**Testing instructions**

1. Navigate to one of your sandboxed sites and clear the browser's local storage.
2. Navigate to the editor
3. The welcome tour should pop up. If not, launch it from the more settings menu.
4. Validate that:
     - [ ] All animations work as expected.
     - [ ] The animations for desktop are visible.

**i18n**

![image](https://user-images.githubusercontent.com/52076348/152013051-fe728e96-d7e1-47c8-aa50-3497a6bd511c.png)


Fixes #56709
